app/views/facilitator/outreach_events/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for :page_header do %>
+  Outreach Events
+<% end %>
+
+<% content_for :page_header_action do %>
+
+    <h4 class="heading mr-1 mb-0">Total<br/>Hours</h4>
+    <div class="pill is-yellow mr-4"><%= @outreach_events.sum(:duration_in_minutes) / 60 %> hours <%= @outreach_events.sum(:duration_in_minutes) % 60 %> minutes</div>
+
+    <%= link_to 'New Outreach Event', new_outreach_event_path, class: "button is-primary" %>
+
+<% end %>
+
+<%= render partial: "facilitator/outreach_events/outreach_events_list", locals: {outreach_events: @outreach_events} %>
```

Comment:
This file is similar enough to `outreach-events/index` that we should reuse it so we only have to maintain one index file.